__tests__/app.test.js

```diff
@@ -240,4 +240,54 @@ describe("#8 GET /api/reviews", () => {
       });
   });
 });
-// 400 passed something not increased votes
+
+describe("#10 POST /api/reviews/:review_id/comments", () => {
+  test.only("201 response returns a posted comment", () => {
+    const objComment = { username: "bainesface", body: "What a great game" };
+    return request(app)
+      .post("/api/reviews/3/comments")
+      .send(objComment)
+      .expect(201)
+      .then(({ body: { addedComment } }) => {
+        expect(addedComment).toEqual(
+          expect.objectContaining({
+            body: expect.any(String),
+            votes: expect.any(Number),
+            author: expect.any(String),
+            review_id: expect.any(Number),
+            created_at: expect.any(String),
+          })
+        );
+      });
+  });
+  test("400 response returns bad request when given an empty object", () => {
+    const objComment = {};
+    return request(app)
+      .post("/api/reviews/3/comments")
+      .send(objComment)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Route not found");
+      });
+  });
+  test("400 response returns bad request when given an invalid object", () => {
+    const objComment = { body: "What a great game" };
+    return request(app)
+      .post("/api/reviews/3/comments")
+      .send(objComment)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Route not found");
```

Comment:
this error message doesn't really make sense - it was the correct route!

try to be as descriptive and helpful as you can with your error messages